src/openms/include/OpenMS/CHEMISTRY/ElementDB.h

```diff
@@ -39,10 +39,10 @@
 #include <OpenMS/DATASTRUCTURES/String.h>
 #include <OpenMS/DATASTRUCTURES/Map.h>
 #include <OpenMS/CHEMISTRY/IsotopeDistribution.h>
+#include <OpenMS/CHEMISTRY/Element.h>
 
 namespace OpenMS
 {
-  class Element;
```

Comment:
Originally I had replaced every double occurence with Element::ContainerType::first_type and Element::ContainerType::second_type but this was not easy to read. Reverting back.